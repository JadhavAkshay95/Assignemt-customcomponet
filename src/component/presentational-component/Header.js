import AppBar from "@material-ui/core/AppBar";
import IconButton from "@material-ui/core/IconButton";
import { makeStyles } from "@material-ui/core/styles";
import Toolbar from "@material-ui/core/Toolbar";
import MenuIcon from "@material-ui/icons/Menu";
import React from "react";
import SearchBoxContainer from "./../container-component/SearchBoxContainer";

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
  },
}));

const Header = () => {
  const classes = useStyles();

  return (
    <>
      <AppBar position="fixed">
        <Toolbar>
          <div className="header-container-dta">
            <IconButton
              edge="start"
              className={classes.menuButton}
              color="inherit"
              aria-label="menu"
            >
              <MenuIcon />
              <span>Mployee</span>
            </IconButton>
            <SearchBoxContainer />
          </div>
        </Toolbar>
      </AppBar>
    </>
  );
};

export default Header;
