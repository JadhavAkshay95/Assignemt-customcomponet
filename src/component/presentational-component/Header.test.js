import { shallow } from "enzyme";
import Header from "./Header";
import React from "react";

describe("Header Component", () => {
  it("Should render header component", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
