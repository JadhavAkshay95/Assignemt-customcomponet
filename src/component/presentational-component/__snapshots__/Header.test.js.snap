// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should render header component 1`] = `
<Fragment>
  <WithStyles(ForwardRef(AppBar))
    position="fixed"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <div
        className="header-container-dta"
      >
        <WithStyles(ForwardRef(IconButton))
          aria-label="menu"
          className="makeStyles-menuButton-2"
          color="inherit"
          edge="start"
        >
          <MenuIcon />
          <span>
            Mployee
          </span>
        </WithStyles(ForwardRef(IconButton))>
        <Connect(SearchBox) />
      </div>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</Fragment>
`;
